lib/bulk-apply-applications.js

```diff
@@ -243,91 +261,166 @@ function bulkApplyApplications (args, callback) {
             done(new Error('Email type not handled for update of applications'));
 
         }
-        return done(null, profiles, event, session, dojo, tickets, emailCode);
+        return done(null, profiles, event, session, dojo, tickets, emailCode, dojoEmailCode);
       });
     }
 
-    function sendEmail (profiles, event, session, dojo, tickets, emailCode, done) {
-      process.nextTick(function () {
-        var emailSubject;
-        if(applications[0].status != 'pending'){
+    //handles creation of payloads and passes them on for sending
+    function sendEmails(profiles, event, session, dojo, tickets, emailCode, dojoEmailCode, done) {
```

Comment:
For some reason (that I don't know) : 
 When as a dojo admin i untick an approved application back to pending, no email is sent.